sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2531,6 +2693,12 @@ export class StorageSharedKeyCredentialPolicy extends CredentialPolicy {
 // @public
 export type SyncCopyStatusType = 'success';
 
+// @public
+export interface Tags {
```

Comment:
This could be `export type Tags = Record<string, string>`, though this is low priority.

Reply:
Updated.